lxd/db/cluster/cluster_groups.go

```diff
@@ -0,0 +1,54 @@
+package cluster
+
+import (
+	"github.com/lxc/lxd/shared/api"
+)
+
+//go:generate -command mapper lxd-generate db mapper -t cluster_groups.mapper.go
+//go:generate mapper reset -i -b "//go:build linux && cgo && !agent"
+//
+//go:generate mapper stmt -e cluster_group objects table=cluster_groups
+//go:generate mapper stmt -e cluster_group objects-by-Name table=cluster_groups
+//go:generate mapper stmt -e cluster_group id table=cluster_groups
+//go:generate mapper stmt -e cluster_group create table=cluster_groups
+//go:generate mapper stmt -e cluster_group rename table=cluster_groups
+//go:generate mapper stmt -e cluster_group delete-by-Name table=cluster_groups
+//go:generate mapper stmt -e cluster_group update table=cluster_groups
+//
+//go:generate mapper method -i -e cluster_group GetMany
+//go:generate mapper method -i -e cluster_group GetOne
+//go:generate mapper method -i -e cluster_group ID
+//go:generate mapper method -i -e cluster_group Exists
+//go:generate mapper method -i -e cluster_group Rename
+//go:generate mapper method -i -e cluster_group Create
+//go:generate mapper method -i -e cluster_group Update
+//go:generate mapper method -i -e cluster_group DeleteOne-by-Name
+
+// ClusterGroup is a value object holding db-related details about a cluster group.
+type ClusterGroup struct {
+	ID          int
+	Name        string
+	Description string   `db:"coalesce=''"`
```

Comment:
Why `coalesce` here? This is a not null field and there are no joins I can see?

Reply:
The original query had it. Actually, it seems we do this for several `description` columns, though it doesn't look like any of those are ever null either. I'll make another PR where I'll clear them all out.

Reply:
Thanks!